apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -424,7 +442,13 @@ class ShareAllowedDialog extends React.Component {
                 )}
                 {publishAllowed && !isPublished && !hasThumbnail && (
                   <div style={{clear: 'both', marginTop: 10}}>
-                    <span style={{fontSize: 12}} className="thumbnail-warning">
+                    <span
+                      style={{
```

Comment:
can you put this style in the style object and also apply it to the other messages below (`restrictedShareInfo` and `downloadReplayVideoButton`)?